config/acm/config.go

```diff
@@ -19,4 +19,17 @@ func Configure(p *config.Provider) {
 		// Deletion takes a while.
 		r.UseAsync = true
 	})
+	p.AddResourceConfigurator("aws_acm_certificate", func(r *config.Resource) {
+		r.LateInitializer = config.LateInitializer{
+			// These are ignored because they conflict with each other.
+			// See the following for more details: https://github.com/upbound/provider-aws/issues/464
+			IgnoredFields: []string{
+				"validation_method",
+				"key_algorithm",
+				"certificate_body",
+				"options.certificate_transparency_logging_preference",
```

Comment:
```suggestion
				"options",
```

When I checked our configuration files, I think it would be more logical to ignore the options configuration block here.

Please see: https://github.com/upbound/provider-aws/blob/9b107dbd15a97e4d38667d5189c9e384b8c5f0ae/config/elasticache/config.go#L30

Reply:
Ok, will fix and ignore options configuration block.